CONTRIBUTING.md

```diff
@@ -8,152 +8,104 @@
+VDK follows the [Code of Conduct](https://github.com/vmware/versatile-data-kit/blob/main/CODE_OF_CONDUCT.md), adapted from the [Contributor Covenant](https://www.contributor-covenant.org/). Please read the Code of Conduct guide to familiarize yourself with the expectations and responsibilities of the community.
 
-Please familiarize yourself with the [Code of Conduct](https://github.com/vmware/versatile-data-kit/blob/main/CODE_OF_CONDUCT.md) before contributing.
+# Contributor License Agreement CLA.
+If you wish to contribute code, you must sign the [contributor license agreement](https://cla.vmware.com/cla/1/preview) (CLA). For any questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq) page.
 
-* _CLA: Before you start working with Versatile Data Kit, please read and sign our Contributor License Agreement [CLA](https://cla.vmware.com/cla/1/preview). If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ]([https://cla.vmware.com/faq](https://cla.vmware.com/faq))._
+# Open Development
+All contributors use GitHub’s pull request (PR) workflow to submit changes. Before merging, all pull requests require two approvals.
 
-## Ways to contribute
+# Semantic Versioning
+VDK follows semantic versioning standards as part of https://semver.org/.
 
-We welcome many different types of contributions and not all of them need a Pull request. Contributions may include:
+Given a version number MAJOR.MINOR.PATCH, increment the:
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards compatible manner
+- PATCH version when you make backwards compatible bug fixes
 
-* New features and proposals
-* Documentation
-* Bug fixes
-* Issue Triage
-* Answering questions and giving feedback
-* Helping to onboard new contributors
-* Other related activities
+# Bugs
 
-# Getting Started
+## Where to Find Known Issues
+Review the [list of existing issues](https://github.com/vmware/versatile-data-kit/issues) to see if the bug has already been reported.
 
-## Contribution Flow
+## Reporting New Issues
+To report a bug, [create a GitHub issue](https://github.com/vmware/versatile-data-kit/issues). Include steps to replicate the bug in the issue description.
 
-This is a rough outline of what a contributor's workflow looks like:
+## Security Bugs
+If you believe you have found a security bug, [create a GitHub issue](https://github.com/vmware/versatile-data-kit/issues) and reach out to us in the [Contacts Us](#contact-us) section.
 
-* Use forks to only contribute changes on examples and documentation.
-   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo as described below.
```

Comment:
nit:
May be missing this info, that would prevent duplicate work around contribution ceremonies.
I saw it is referred to in `## Submit a pull request` section - however the statements differ.

Reply:
Moved this to a note in Development Workflow section.